packages/snscrape/PKGBUILD

```diff
@@ -0,0 +1,33 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgname=snscrape
+pkgver=v0.4.3.20220106.r73.gcbdaee6
+pkgrel=1
+pkgdesc='A social networking service scraper in Python.'
+arch=('any')
+groups=('blackarch' 'blackarch-recon' 'blackarch-social')
+url='https://github.com/JustAnotherArchivist/snscrape'
+license=('GPL3')
+depends=('libxml2' 'libxslt' 'python' 'python-lxml')
```

Comment:
Looks to me that you are missing the python deps https://github.com/JustAnotherArchivist/snscrape/blob/cbdaee6864aa8d3e9e4326d4fe224a517aabbc2c/setup.py#L27-L31

Also it don't looks like all othe one you puts are needed.

Reply:
Added needed dependencies: 'python-filelock' 'python-lxml' 'python-requests' 'python-beautifulsoup4'

I didn't add pytz due to apply only to < Python 3.9.

According to https://pypi.org/project/snscrape/#description , lxml needs libxslt and libxml2. I removed libxslt because already installed by python-lxml package, and removed also libxml2 because installed when libxslt is installed.